Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/BackupIntroViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  BackupIntroViewController.swift
+//  Multisig
+//
+//  Created by Vitaly on 11.04.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class BackupIntroViewController: UIViewController {
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var messageLabel: UILabel!
+    @IBOutlet weak var tipsView: TipsView!
+    @IBOutlet weak var backupButton: UIButton!
+    @IBOutlet weak var cancelButton: UIButton!
+    
+    var backupCompletion: (_ backup: Bool) -> Void = { _ in }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationItem.title = "Back up key"
+        
+        titleLabel.setStyle(.primary)
+        messageLabel.setStyle(.secondary)
+        tipsView.setContent(
+            title: "Security tips",
+            tips: [
+                "Never share your seed phrase with anyone!",
+                "Write it down on paper or keep it in a vault.",
+                "Store it in a secret place or multiple places that you trust."
+            ]
+        )
+        
+        backupButton.setText("Back up manually", .filled)
+        cancelButton.setText("Not now", .plain)
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+
+        if self.isMovingFromParent {
+            backupCompletion(false)
```

Comment:
Can you elaborate more on what is the intent of these lines?

Reply:
If user goes back from backup intro screen, it would lead back to enter key name screen. Key is already added and attempt to enter something and save it again would lead to exception. This prevents navigation back to enter key name screen and treats back navigation as dismissal.

Reply:
A, ok. I think it was mentioned but never written down or changed the UI: we should disable the back button in this screen because user should not be able to go back to editing name after that is done. You can double-check with @sche .

Reply:
user can still swipe back even without back button. This code will handle it.

Reply:
the swipe gesture can be disabled. If it's not on the 'create passcode' screen as well, then we should change it.